### PR TITLE
Different CI

### PR DESCRIPTION
**Pull request by Hoverbear**
Use not-Nix so we can have not-sandboxed tests so we can test random user crates etc easier.

### CODE REVIEW

**Comment by Hoverbear** [comment]
Hey @BradyBonnette would you be kind enough to figure out if I did an ok job porting your ci script?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,76 +1,155 @@
...
+      uses: actions/cache@v2
+      continue-on-error: false
+      with:
+        path: |
+          ~/.cargo/bin/
+          ~/.cargo/registry/index/
+          ~/.cargo/registry/cache/
+          ~/.cargo/git/db/
+          target/
+        key: v0-${{ runner.os }}-${{ hashFiles('**/Cargo.lock', '.github/workflows/tests.yml') }}
```
Comment (BradyBonnette) [inline .github/workflows/ci.yml:128]: @Hoverbear 

Will `v0` here in the cache key need to change in the future?

**Review by BradyBonnette — COMMENTED** [review]
Everything looks good, though I had a question regarding one tiny thing (see inline comment regarding cache key)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Hoverbear) [reply .github/workflows/ci.yml:128]: If we change the cache layout, yes! That's its purpose!

Comment (BradyBonnette) [reply .github/workflows/ci.yml:128]: Ah, I see. Works for me :)

**Review by BradyBonnette — APPROVED** [review]

**Comment by workingjubilee** [comment]
Seems good to me~!

**Pull request merged**